01_run_asli_calculations.sh

```diff
@@ -9,6 +9,6 @@ echo "Requesting with the following arguments: $DATA_ARGS_ERA5".
 asli_data_era5 $DATA_ARGS_ERA5
```

Comment:
I wonder if including `set -e` at the top of these scripts might be helpful. I know it's at the top of the top-level script, but I'm unsure on the finer points of the behaviour, e.g. when calling a script from another. Might be a bit belt-and-braces. Having it there would also protect against failures in the case that any of the scripts are run in isolation.

Reply:
Agree this is worth the duplication, as some might be ran individually